### PR TITLE
Initial LighthouseCI Commit

### PR DESCRIPTION
**Pull request by unlikelyzero**
Closes https://github.com/nasa/openmct/issues/3939

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,4 +40,7 @@ npm-debug.log
 # karma reports
 report.*.json
 
-package-lock.json
+# Lighthouse reports
+.lighthouseci
+
+package-lock.json
```
Comment (nikhilmandlik) [inline .gitignore:46]: add new line at end of file

**Review by nikhilmandlik — DISMISSED** [review]
small change, LGTM

**Comment by unlikelyzero** [comment]
@nikhilmandlik resolved, thanks!

File: .github/workflows/lighthouse.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        required: false
+        default: 'master' 
+jobs:
+  lighthouse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+      - uses: actions/setup-node@v1
```
Comment (shefalijoshi) [inline .github/workflows/lighthouse.yml:16]: Looks good as long as we're installing a minimum of version 10 of node.

**Review by shefalijoshi — APPROVED** [review]

File: .github/workflows/lighthouse.yml (hunk 2 above)
Comment (unlikelyzero) [reply .github/workflows/lighthouse.yml:16]: https://github.com/nasa/openmct/issues/3952

**Pull request merged**